curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/introduction-to-data-analysis.md

```diff
@@ -10,29 +10,29 @@ dashedName: introduction-to-data-analysis
 
 More resources:
 
-\- [Slides](https://docs.google.com/presentation/d/1fDpjlyMiOMJyuc7_jMekcYLPP2XlSl1eWw9F7yE7byk)
+\- [Slides](https://docs.google.com/presentation/d/1cUIt8b2ySz-85_ykfeuuWsurccwTAuFPn782pZBzFsU/edit?usp=sharing)
 
 # --question--
 
 ## --text--
 
-Why should you choose R over Python for data analysis?
+Which of the following is not part of Data Analysis?
```

Comment:
Do we want to bold **not**?

Reply:
I have applied a commit to make it bold.